miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -713,8 +714,9 @@ HotTarget.sample = (function() {
                 })) {
                   targets.push({
                     name: "Library",
-                    action: function(replicates) {
+                    action: function(replicates, newBoxId) {
                       var params = {
+                        boxId: newBoxId,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)